compose.yml

```diff
@@ -184,32 +188,28 @@ services:
 
     updateEFO:
         image: python:3.7
-        env_file:
-            - ./.env
-            - ./.env.secret.updateEFO
         restart: unless-stopped
         profiles:
             - production
         volumes:
             - ./updateEFO:/updateEFO
             - ./_logs/updateEFO:/var/log/updateEFO
             - ./updateEFO/rotatelogs-compress.sh:/rotatelogs-compress.sh
+        secrets:
+            - efo_secrets
```

Comment:
Noting that just mounting the secrets file won't do anything by itself aside from mounting the secret as a file into `/run/secrets`. Something still need to read this file and ensure the values from the file are propagated to the configuration. 

In case of the `updateEFO` there are envs like `MW_BOT_USER` , `MW_BOT_PASSWORD` and `MW_UPDATE_EFO_PAUSE` in use. This case also nicely highlights the reason why we're getting rid of the `env_file` usage - with `env_file` it's challenging to list the variables needed on the Compose file due to various limitations applied by the Compose https://docs.docker.com/compose/how-tos/environment-variables/set-environment-variables/#use-the-env_file-attribute